pulp_file/tests/functional/api/test_sync.py

```diff
@@ -28,6 +28,45 @@ class BasicFileSyncTestCase(unittest.TestCase):
     def setUpClass(cls):
         """Create class-wide variables."""
         cls.cfg = config.get_config()
+        cls.client = api.Client(cls.cfg, api.json_handler)
+
+    def test_file_decriptors(self):
+        """Test whether file descriptors are closed properly.
+
+        This test targets the following issues:
+
+        `Pulp #4073 <https://pulp.plan.io/issues/4073>`_
+
+        Do the following:
+
+        1. Create and sync a repo.
+        2. Install ``lsof``, and issue a command to verify that files in the
+           path ``/var/lib/pulp/`` are closed after the sync.
+        3. Assert that issued command returns `0` opened files.
+        """
+        pkg_mgr = cli.PackageManager(self.cfg)
+        try:
+            pkg_mgr.name
+        except NoKnownPackageManagerError:
+            raise unittest.SkipTest('This test requires dnf or yum.')
+
+        pkg_mgr.install('lsof')
+        self.addCleanup(pkg_mgr.uninstall, 'lsof')
+
+        repo = self.client.post(REPO_PATH, gen_repo())
+        self.addCleanup(self.client.delete, repo['_href'])
+
+        body = gen_file_remote()
+        remote = self.client.post(FILE_REMOTE_PATH, body)
+        self.addCleanup(self.client.delete, remote['_href'])
+
+        sync(self.cfg, remote, repo)
+
+        cli_client = cli.Client(self.cfg, cli.echo_handler)
+        path = '/var/lib/pulp/'
+        cmd = 'lsof -t +D {}'.format(path).split()
```

Comment:
```suggestion
        cmd = 'lsof -t +D {}'.format('/var/lib/pulp/').split()
```

To avoid unneccessary memory allocation of path? is that used anywhere else?